echopype/echodata/sensor_ep_version_mapping/v05x_to_v06x.py

```diff
@@ -0,0 +1,1091 @@
+    ``beamwidth_twoway_alongship/athwartship``,
+    ``angle_athwartship/alongship``
+
+    Parameters
+    ----------
+    ed_obj : EchoData
+        EchoData object that was created using echopype version 0.5.x.
+    sensor : str
+        Variable specifying the sensor that created the file.
+
+    Notes
+    -----
+    The function directly modifies the input EchoData object.
+
+    """
+
+    if sensor in ["EK60", "EK80"]:
+
+        for beam_group in ed_obj._tree["Sonar"].children:
+            beam_group.ds.beamwidth_twoway_alongship.attrs["comment"] = (
+                "Introduced in echopype for Simrad echosounders to avoid "
+                "potential confusion with convention definitions. The alongship "
+                "angle corresponds to the minor angle in SONAR-netCDF4 vers 2. The "
+                "convention defines one-way transmit or receive beamwidth "
+                "(beamwidth_receive_minor and beamwidth_transmit_minor), but Simrad "
+                "echosounders record two-way beamwidth in the data."
+            )
+
+            beam_group.ds.beamwidth_twoway_athwartship.attrs["comment"] = (
+                "Introduced in echopype for Simrad echosounders to avoid "
+                "potential confusion with convention definitions. The athwartship "
+                "angle corresponds to the major angle in SONAR-netCDF4 vers 2. The "
+                "convention defines one-way transmit or receive beamwidth "
+                "(beamwidth_receive_major and beamwidth_transmit_major), but Simrad "
+                "echosounders record two-way beamwidth in the data."
+            )
+
+            beam_group.ds.angle_offset_alongship.attrs["comment"] = (
+                "Introduced in echopype for Simrad echosounders. The alongship "
+                "angle corresponds to the minor angle in SONAR-netCDF4 vers 2. "
+            )
+
+            beam_group.ds.angle_offset_athwartship.attrs["comment"] = (
+                "Introduced in echopype for Simrad echosounders. The athwartship "
+                "angle corresponds to the major angle in SONAR-netCDF4 vers 2. "
+            )
+
+            beam_group.ds.angle_sensitivity_alongship.attrs[
+                "comment"
+            ] = beam_group.ds.angle_offset_alongship.attrs["comment"]
+
+            beam_group.ds.angle_sensitivity_athwartship.attrs[
+                "comment"
+            ] = beam_group.ds.angle_offset_athwartship.attrs["comment"]
+
+            if "angle_alongship" in beam_group.ds:
+                beam_group.ds.angle_alongship.attrs[
+                    "comment"
+                ] = beam_group.ds.angle_offset_alongship.attrs["comment"]
+
+            if "angle_athwartship" in beam_group.ds:
+                beam_group.ds.angle_athwartship.attrs[
+                    "comment"
+                ] = beam_group.ds.angle_offset_athwartship.attrs["comment"]
+
+
+def _beam_groups_to_convention(ed_obj, set_grp_cls):
```

Comment:
May be a good idea to make this `_beamgroups_to_convention` to be consistent with what you have in `set_grp_cls.beamgroups_to_convention`

Reply:
Are you suggesting that I rename this function to `_beamgroups_to_convention`?

Reply:
I just think it should be consistent in all places. My inclination is `_beam_groups_*` because the groups are called `Beam_groupX`.

Reply:
Ok, I will change `set_grp_cls.beamgroups_to_convention` to `set_grp_cls.beam_groups_to_convention`.